modules/diskq/qdisk.c

```diff
@@ -558,6 +558,13 @@ _load_state(QDisk *self, GQueue *qout, GQueue *qbacklog, GQueue *qoverflow)
                evt_tag_str("filename", self->filename),
                evt_tag_int("queue_length", self->hdr->length),
                evt_tag_int("size", self->hdr->write_head - self->hdr->read_head));
+
+      msg_debug("Reliable disk-buffer internal state",
+                evt_tag_str("filename", self->filename),
+                evt_tag_int("backlog_head", self->hdr->backlog_head),
+                evt_tag_int("read_head", self->hdr->read_head),
```

Comment:
Please use evt_tag_long.
evt_tag_int uses `"%d"`, so it might not work with gint64.

Reply:
Nice catch, I updated in other messages also (in disk buffer).